app/components/ui/Tooltip.tsx

```diff
@@ -1,79 +1,26 @@
-  arrowClassName?: string;
-  tooltipStyle?: React.CSSProperties;
-  position?: 'top' | 'bottom' | 'left' | 'right';
-  maxWidth?: number;
-  delay?: number;
+export interface TooltipProps {
+  children: ReactNode;
+  content: ReactNode;
 }
 
-const WithTooltip = forwardRef(
-  (
-    {
-      tooltip,
-      children,
-      sideOffset = 5,
-      className = '',
-      arrowClassName = '',
-      tooltipStyle = {},
-      position = 'top',
-      maxWidth = 250,
-      delay = 0,
-    }: TooltipProps,
-    _ref: ForwardedRef<HTMLElement>,
-  ) => {
-    return (
-      <Tooltip.Root delayDuration={delay}>
-        <Tooltip.Trigger asChild>{children}</Tooltip.Trigger>
-        <Tooltip.Portal>
-          <Tooltip.Content
-            side={position}
-            className={`
-              z-[2000]
-              px-2.5
-              py-1.5
-              max-h-[300px]
-              select-none
-              rounded-md
-              bg-bolt-elements-background-depth-3
-              text-bolt-elements-textPrimary
-              text-sm
-              leading-tight
-              shadow-lg
-              animate-in
-              fade-in-0
-              zoom-in-95
-              data-[state=closed]:animate-out
-              data-[state=closed]:fade-out-0
-              data-[state=closed]:zoom-out-95
-              ${className}
-            `}
-            sideOffset={sideOffset}
-            style={{
-              maxWidth,
-              ...tooltipStyle,
-            }}
+export default function Tooltip({ children, content }: TooltipProps) {
+  return (
+    <TooltipPrimitive.Provider>
+      <TooltipPrimitive.Root>
+        <TooltipPrimitive.Trigger asChild>{children}</TooltipPrimitive.Trigger>
+        <TooltipPrimitive.Portal>
+          <TooltipPrimitive.Content
+            className="z-50 overflow-hidden rounded-md border border-bolt-elements-borderColor bg-bolt-elements-background-depth-2 px-3 py-1.5 text-sm text-bolt-elements-text-color animate-in fade-in-0 zoom-in-95 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2"
+            sideOffset={5}
           >
-            <div className="break-words">{tooltip}</div>
-            <Tooltip.Arrow
-              className={`
-                fill-bolt-elements-background-depth-3
-                ${arrowClassName}
-              `}
-              width={12}
-              height={6}
-            />
-          </Tooltip.Content>
-        </Tooltip.Portal>
-      </Tooltip.Root>
-    );
-  },
-);
-
-export default WithTooltip;
+            {content}
+            <TooltipPrimitive.Arrow className="fill-bolt-elements-borderColor" />
+          </TooltipPrimitive.Content>
+        </TooltipPrimitive.Portal>
+      </TooltipPrimitive.Root>
+    </TooltipPrimitive.Provider>
+  );
+}
```

Comment:
i think its best to keep the name and argument names for this should remain same so that we don't need to update the code everywhere its used.
it creates less merge conflicts with others who are already working with other stuff and also less confuses people who are learning the codebase and different component of it